build/drand.go

```diff
@@ -11,6 +12,9 @@ type DrandEnum int
 func DrandConfigSchedule() dtypes.DrandSchedule {
 	out := dtypes.DrandSchedule{}
 	for start, network := range DrandSchedule {
+		if os.Getenv("LOTUS_DISABLE_DRAGON") == "1" && network == DrandQuicknet {
```

Comment:
Filter out `start < 0`? We always set disabled upgrade heights to 0.

Reply:
Gah!!! we should be setting those to -1, not infinity. Oh well...

Reply:
too late cause it gets set as `3817920+120` before this, or init gets called